### PR TITLE
RAI-468: Use trustwallet assets as a fallback

### PR DESCRIPTION
**Pull request by Kureev**

### CODE REVIEW

File: src/components/transaction-list/TransactionList.js [hunk 1]
```diff
@@ -83,8 +89,8 @@ class TransactionList extends React.PureComponent {
     const addressOrEns = accountENS || this.formatAddress(accountAddress);
 
     const data = {
-      requests,
-      transactions,
+      requests: requests.map(this.checksumAddress),
```
Comment (brunobar79) [inline src/components/transaction-list/TransactionList.js:92]: I don't think you need this for requests

File: ios/TransactionListViewCell.swift [hunk 2]
```diff
@@ -43,6 +43,9 @@ class TransactionListViewCell: TransactionListBaseCell {
     if transaction.symbol != nil {
       if let img = UIImage.init(named: transaction.symbol.lowercased()) {
         coinImage.image = img
+      } else if transaction.address != nil {
+        let url = URL(string: "https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/ethereum/assets/\(transaction.address!)/logo.png");
+        coinImage.sd_setImage(with: url)
```
Comment (brunobar79) [inline ios/TransactionListViewCell.swift:48]: There's one more thing left. On the JS side, we have a listener for the image onError event, and in that case we fallback to the case that you have below: See https://github.com/rainbow-me/rainbow/blob/develop/src/components/coin-icon/CoinIcon.js#L71

File: src/components/transaction-list/TransactionList.js (hunk 1 above)
Comment (brunobar79) [reply src/components/transaction-list/TransactionList.js:92]: also @jinchung , can you confirm if we can checksum the address at the transaction parser level and prevent this additional loop?

Comment (Kureev) [reply src/components/transaction-list/TransactionList.js:92]: If it is possible, I'd love to see it done on the backend side so we don't have to apply any processing to the data. Moreover, would be great to compute a transaction state there as well. As of now, we have to operate transaction.status and transaction.type in order to derive the actual transaction state. We implement this logic on both front-end and back-end and it seems unnecessary complex. Anyway, let me know what you think

File: ios/TransactionListViewCell.swift (hunk 2 above)
Comment (Kureev) [reply ios/TransactionListViewCell.swift:48]: Oh, I see! You're right, we need to handle the fetch error for sure. Will fix it ASAP

File: src/components/transaction-list/TransactionList.js (hunk 1 above)
Comment (jinchung) [reply src/components/transaction-list/TransactionList.js:92]: Sure - I'll take care of these two things: checksumAddress and transaction state

Comment (jinchung) [reply src/components/transaction-list/TransactionList.js:92]: @brunobar79 If all else looks good to you, you can merge this into Christian's branch and I'll create a separate ticket / PR

**Pull request merged**